helm/config/images.env

```diff
@@ -0,0 +1,10 @@
+RAY_OPERATOR_IMG_REPO=syntho.azurecr.io/syntho-ray-operator
+RAY_OPEARATOR_IMG_TAG=latest
+RAY_IMAGE_IMG_REPO=syntho.azurecr.io/syntho-ray
+RAY_IMAGE_IMG_TAG=0.49.0-cpu
```

Comment:
Same here: we could set these as latest for the default right? Or are they overwritten in all cases?

Reply:
I'd say we should keep those versions strict when considering versioning of the entire stack. Think like this, 

If we make it `latest` and create versions like `0.1.0` for the entire stack, that 0.1.0 versions' components will change constantly if backend/frontend/core has new changes that are tagged with again `latest` tag.

I tested all the components with the given versions and I am now sure that this stack is going to be deployed successfully. And, I'd like to tag a release as `beta-0.1.0` so that from now on, `beta-0.1.0` release will always work. But it we'd change it `latest` then there is a potential that `beta-0.1.0` might break things when a broken `backend` is tagged as `latest`.

Reply:
WDYT @Younday ?